client/src/pages/Home.js

```diff
@@ -13,5 +13,17 @@ export default function Home() {
     history.push(`/map`);
   }
 
-  return <SubmitForm handleOnSubmit={handleOnSubmit} />;
+  return (
+    <>
+      <div className="home__banner__wrapper">
+        <h2 className="home__banner__title1">Finde einen</h2>
+        <h2 className="home__banner__title2">Spielplatz</h2>
+        <h2 className="home__banner__title3">In deiner Nähe</h2>
+      </div>
```

Comment:
BEM only has 1x __
Block__element--modifier

here's a nice article relating to BEM in React:
https://medium.com/@seanmcp/how-and-when-to-use-bem-in-react-edabad2b805a